hibernate-rx-core/src/main/java/org/hibernate/rx/service/RxGenerationTarget.java

```diff
@@ -0,0 +1,41 @@
+package org.hibernate.rx.service;
+
+import org.hibernate.rx.service.initiator.RxConnectionPoolProvider;
+import org.hibernate.service.ServiceRegistry;
+import org.hibernate.tool.schema.internal.exec.GenerationTarget;
+
+/**
+ * Adaptor that redirects DDL generated by the schema export
+ * tool to the reactive connection.
+ *
+ * @author Gavin King
+ */
+public class RxGenerationTarget implements GenerationTarget {
+	RxConnection connection;
+	private ServiceRegistry registry;
+
+	public RxGenerationTarget(ServiceRegistry registry) {
+		this.registry = registry;
+	}
+
+	@Override
+	public void prepare() {
+		connection = registry.getService( RxConnectionPoolProvider.class ).getConnection();
+	}
+
+	@Override
+	public void accept(String command) {
+		try {
+			connection.preparedQuery(command).toCompletableFuture().join();
```

Comment:
@vietj @FroMage @Sanne @DavideD WDYT of this?

I want to run Schema Export against the Vert.x driver, but I don't need it to actually be reactive. Also, I don't want to rewrite all of Hibernate's schema export stuff to use `CompletionStage` or whatever.

This seems to work fine, *except* with the `@RunWith(VertxUnitRunner.class)` where sometimes I get a harmless stacktrace from Vert.x at the end of my test.

Is this for some reason a terrible thing to be doing? Is there some other way we should be doing it?

Reply:
@gavinking What's the stack trace complaining about? Is it the Vertx blocked thread checker?  If that's the case, I think we can safely ignore that because we are intending to be blocking here. To silence the blocked thread checker, we could use our own `Vertx` instance where we can set the blocked thread checker interval to a really high value like so:
```java
    VertxOptions vertxOptions = new VertxOptions();
    vertxOptions.setBlockedThreadCheckInterval(600);
    vertxOptions.setBlockedThreadCheckIntervalUnit(TimeUnit.MINUTES);
    Vertx vertx = Vertx.vertx(vertxOptions);
```

Reply:
Yeah it looks to me like Vert.x shuts down the thread because it thinks it's misbehaving.

Let's try your suggestion. Thanks.

Reply:
@aguibert so I looked for a place to set this up in `VertxUnitRunner` and didn't find anything obvious. @vietj any suggestion?